src/datastore-service.js

```diff
@@ -179,7 +179,7 @@ function createDataStoreService(connectionPool: any): DataStoreService {
       record: record.toString(),
       parentId: RecordUtils.parseParentId(record),
       timestamp: now,
-      recordTimestamp: moment(record.get(/^005$/).shift().value, 'YYYYMMDDHHmmssSS').format('YYYY-MM-DDTHH:mm:ss.SS')
+      recordTimestamp: moment(_.chain(record.get(/^005$/)).shift().value, 'YYYYMMDDHHmmssSS').format('YYYY-MM-DDTHH:mm:ss.SS')
```

Comment:
Without `_.chain` Flow reports and error:
```sh
property `value`. Property not found in
```

I don't know if there's a better way to solve this problem (Flow type definition is a union consisting of ControlField and DataField).